### PR TITLE
Blog Post: Hacktoberfest 2024

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Details: https://github.com/wildfly/wildfly.org/issues/654

### CODE REVIEW

File: _posts/2024-09-20-Hacktoberfest-2024.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+. *WildFly:* WildFly is a powerful, modular, and lightweight application server that empowers you to build amazing applications.
+* Here are the https://github.com/wildfly/wildfly[GitHub repository], https://github.com/wildfly/wildfly/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/issues/?filter=12421535[Hacktoberfest Issues link].
+
+. *WildFly Elytron:* WildFly Elytron is a set of APIs and SPIs for application server security and client side security. Within the WildFly server, WildFly Elytron is used to secure applications deployed to the server as well as management access to the server. It is a standalone library that can also be used in other Java server environments.
+* Here are the https://github.com/wildfly-security/wildfly-elytron[GitHub repository], https://github.com/wildfly-security/wildfly-elytron/blob/2.x/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/issues/?filter=12383825[Hacktoberfest Issues link].
```
Comment (fjuma) [inline _posts/2024-09-20-Hacktoberfest-2024.adoc:38]: Sorry, I realized had the wrong filter listed in the spreadsheet. Please use https://issues.redhat.com/issues/?filter=12364234. Thanks!

File: _posts/2024-09-20-Hacktoberfest-2024.adoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+. *WildFly GitHub Bot:* WildFly GitHub Bot is a small easy-to-start application that we utilize to manage the workflow of WildFly's Pull Requests. It, for instance, requires reviews from relevant reviewers or sets labels based on the PR's state. We have a lot of ideas that you can start implementing.
+* Here are the https://github.com/wildfly/wildfly-github-bot[GitHub repository], https://github.com/wildfly/wildfly-github-bot/blob/main/CONTRIBUTING.md[Contributing Guide], and https://github.com/wildfly/wildfly-github-bot/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22[Hacktoberfest Issues link].
+
+. *WildFly Docker Images:* The WildFly Docker project provides Docker images for running the WildFly application server with support for various JDK versions. It allows easy customization of server configurations, application deployment, and management in both standalone and domain modes.
```
Comment (jmesnil) [inline _posts/2024-09-20-Hacktoberfest-2024.adoc:62]: LGTM

**Review by jmesnil — APPROVED** [review]

**Review by fjuma — APPROVED** [review]
Thanks @RanabirChakraborty! The Elytron section looks good!

**Review by hpehl — APPROVED** [review]
Thanks @RanabirChakraborty 
All ok with the PatternFly Java section.

File: _posts/2024-09-20-Hacktoberfest-2024.adoc [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+* Here are the https://github.com/ansible-middleware[GitHub repositories], https://github.com/ansible-middleware/keycloak/blob/main/CONTRIBUTING.md[Contributing Guide], and https://github.com/search?q=org%3Aansible-middleware+state%3Aopen+label%3Ahacktoberfest&type=issues[Hacktoberfest Issues link].
+
+. *Undertow:* Undertow is a versatile and high-performance web server that provides both blocking and non-blocking APIs based on NIO. Its modular architecture allows you to build a web server tailored to your needs, from a full Java EE servlet container to a low-level non-blocking handler.
+* Here are the https://github.com/undertow-io/undertow[GitHub repository], https://github.com/undertow-io/undertow/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/issues/?filter=12422070[Hacktoberfest Issues link].
+
+. *WildFly Core:* WildFly Core provides the core runtime that is used by the Wildfly application server. This includes Modular class loading, Unified management including domain mode, Basic deployment architecture, and CLI for management.
```
Comment (bstansberry) [inline _posts/2024-09-20-Hacktoberfest-2024.adoc:47]: Wildfly -> WildFly

I sent a PR to WildFly Core to fix that in its README.

Also, in the second sentence I think everything should be lower case except, of course,"This" and "CLI".

Comment (bstansberry) [reply _posts/2024-09-20-Hacktoberfest-2024.adoc:47]: Also

"and CLI" -> "and a CLI"

Comment (RanabirChakraborty) [reply _posts/2024-09-20-Hacktoberfest-2024.adoc]: Thanks! done

**Review by bstansberry — APPROVED** [review]

File: _posts/2024-09-20-Hacktoberfest-2024.adoc [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+. *Narayana:* Narayana is a transactions toolkit that provides support for applications using various standards-based transaction protocols. It's an essential component of the WildFly application server and continues to be developed as a standalone transaction manager.
+* Here are the https://github.com/jbosstm/narayana[GitHub repository], https://github.com/jbosstm/narayana/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/issues/?filter=12421681[Hacktoberfest Issues link].
+
+. *Intersmash:* Intersmash is a Java library that makes it easy to automate the provisioning and execution of tests in cloud-native environments. It helps the user prototype and test complex interoperability scenarios on kubernetes compliant cloud-native environments and platforms, most notably OpenShift. (Other Kubernetes implementations will be supported in the future.)
+* Here are the https://github.com/Intersmash/intersmash[GitHub repository], https://github.com/Intersmash/intersmash/blob/main/CONTRIBUTING.md[Contributing Guide], and https://github.com/Intersmash/intersmash/issues[Hacktoberfest Issues link].
```
Comment (fabiobrz) [inline _posts/2024-09-20-Hacktoberfest-2024.adoc:54]: Maybe https://github.com/Intersmash/intersmash/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest to have a view of just the issues which are related to Hacktoberfest

**Review by fabiobrz — COMMENTED** [review]
Thanks @RanabirChakraborty for this PR! I've got just one comment about the Intersmash section, specifically the issues list. But definitely not blocking, in case yo'd prefer to provide a list of al the issues.

File: _posts/2024-09-20-Hacktoberfest-2024.adoc (hunk 4 above)
Comment (RanabirChakraborty) [reply _posts/2024-09-20-Hacktoberfest-2024.adoc]: Thanks! Done.

**Review by fabiobrz — APPROVED** [review]
Thanks @RanabirChakraborty, LGTM!

**Comment by bstansberry** [comment]
Thanks @RanabirChakraborty !

**Pull request merged**